testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/appclient/util/AppClientWrapper.java

```diff
@@ -202,6 +206,8 @@ private String getAppClientCommand() throws Exception {
                 " -Djboss.server.base.dir="+ asDist + "/appclient" +
                 " -Djboss.home.dir="+ asDist +
                 " " + this.appClientArgs + " " + archiveArg + " " + args;
```

Comment:
`args` is the value used for AppClientSubsystemResourceDefinition.PARAMETERS, but I don't see it wrapped by an expression, shouldn't it be?

Reply:
Thanks @yersan -- I missed that that one supported expression. One test method passes a non-empty arg so I'll wrap that value in an expression.

Reply:
I've updated AbstractSimpleApplicationClientTestCase.simpleAppClientTest().